Update Getting Started and related pages (#6717)
w3f/polkadot-wiki: docs/general/getting-started.md

## What is Polkadot?

> !!!info "Explore Polkadot with [secure and user-friendly wallets](https://www.polkadot.network/ecosystem/wallets/)."

Reviewer: Maybe make it more general, i.e., "user-friendly apps".

> Polkadot is a **blockchain-powered multi-core decentralized cloud computer**. Let's break it down bit by bit:

Reviewer: ```suggestion
Polkadot is a **blockchain-based protocol that facilitates multi-core, decentralized cloud computation**. Let's break it down bit by bit:
```

> - **blockchain-powered:** Polkadot at its heart has a blockchain that permanently and immutably stores sensitive information about the protocol and applications running on top of it

Reviewer: ```suggestion
- **blockchain-based:** Polkadot at its heart has a blockchain that permanently and immutably stores sensitive information about the protocol and applications running on top of it
```

- **multi-core:** Like traditional computers, Polkadot has a virtual CPU with multiple virtual cores

> - **decentralized cloud computer:** Instead of being run by a single entity, Polkadot is run by real computers owned by different operators scattered around the globe, part of a decentralized peer-to-peer network

Reviewer: ```suggestion
- **decentralized cloud computer:** Instead of being run by a single entity, Polkadot is run by real computers owned by different operators (validators) scattered around the globe, part of a decentralized peer-to-peer network
```

> **Polkadot is a Web3 service provider**: it provides secure and resilient computation and interoperability for any application connected to its virtual cores. Applications can be fully-fledged blockchains, known as [parachains](../learn/learn-parachains.md) or Polkadot rollups. Instead of bootstrapping their security, rollups can connect to Polkadot via its virtual cores by buying [coretime](../learn/learn-agile-coretime.md) with DOT and tap into Polkadot's multi-billion shared security. By doing so, they inherently access Polkadot's secure interoperability and can communicate with other applications running on different cores of Polkadot's virtual CPU.

Reviewer: ```suggestion
**Polkadot is a Web3 service provider**: it provides secure and resilient computation and interoperability for any application connected to its network. Applications can be fully-fledged blockchains deployed as rollups on Polkadot (also known as [parachains](../learn/learn-parachains.md)). Instead of bootstrapping their security, rollups can connect to Polkadot via its virtual cores by buying [coretime](../learn/learn-agile-coretime.md) with DOT and tap into Polkadot's multi-billion shared security. By doing so, they inherently access Polkadot's secure interoperability and can communicate with other applications running on different cores of Polkadot's virtual CPU.
```

> Polkadot decentralized computer is owned by the [DOT token](../learn/learn-DOT.md) holders who can shape its future through on-chain [governance](../learn/learn-polkadot-opengov.md) (also called Polkadot

Reviewer: ```suggestion 
Polkadot is owned by the [DOT token](../learn/learn-DOT.md) holders who can shape its future through on-chain [governance](../learn/learn-polkadot-opengov.md) (also called Polkadot
```

- [The Polkadot Blockchain Academy X (PBA-X)](https://pbax.polkadot.academy/): an online course about cryptography, blockchain, crypto-economics, the Polkadot protocol and Polkadot SDK.
- [The Polkadot Blockchain Academy Campus (PBA Campus)](https://polkadot.academy/pba-campus/): a three-week intensive course that will make you a blockchain core developer or a dApp developer.

## Tutorials for Developers

- [Rust and Polkadot Workshop](https://www.shawntabrizi.com/rust-and-polkadot-workshop/#/): A workshop to teach new students the basics of Blockchains, Rust, and the Polkadot SDK.
- [DotCodeSchool](https://dotcodeschool.com): An interactive online school that teaches you how to build meaningful web3 applications using the Polkadot SDK.
- [Polkadot Study](https://polkadot.study/): A platform where developers can write tutorials for
  the Polkadot ecosystem. Part of the development of the platform was funded by the
  [Kusama treasury](https://kusama.subsquare.io/referenda/102). The platform also hosts
  [Substrate in Bits](https://polkadot.study/tutorials/substrate-in-bits/), a technical content
  series aimed at solving the pain points of developers building with

>   [Substrate](https://docs.polkadot.com/develop/parachains/intro-polkadot-sdk/#substrate) and [Rust](https://www.rust-lang.org/).

Reviewer: ```suggestion
  [Polkadot SDK (Substrate)](https://docs.polkadot.com/develop/parachains/intro-polkadot-sdk/#substrate) and [Rust](https://www.rust-lang.org/).
```
